assets/src/components/story-block-drop-zone.js

```diff
@@ -52,6 +53,8 @@ class BlockDropZone extends Component {
 			positionLeft: getPercentageFromPixels( 'x', clonePosition.left - wrapperPosition.left ),
 			positionTop: getPercentageFromPixels( 'y', clonePosition.top - wrapperPosition.top ),
 		} );
+		// Make sure the block stays selected. // @todo This is not working.
+		selectBlock( srcClientId );
```

Comment:
@swissspidy Quick question: do you remember if deselecting the block after dragging and after rotating was something that you purposefully added when implementing the rotation? The block gets deselected in both cases, it's not clear why (`selectBlock` here doesn't work here). I'm asking just in case you added it purposefully, otherwise, I'll continue looking for the reason.

Reply:
Indeed it was on purpose, see https://github.com/ampproject/amp-wp/issues/2211#issuecomment-487906591.

See `clearSelectedBlock` in `stopBlockRotation` in `assets/src/components/higher-order/with-amp-story-settings.js`

Reply:
Good to know, thanks!